lib/Preview/EPubPreview.php

```diff
@@ -51,22 +51,13 @@ public function getMimeType(): string {
 		return '/^application\/epub\+zip$/';
 	}
 
-	/**
-	 * {@inheritDoc}
-	 */
-	public function isAvailable(FileInfo $file): bool {
-		return $file->getSize() > 0;
-	}
```

Comment:
Make sure to add that one back because `isAvailable()` returns `true` by default in `ProviderV2`.

Reply:
I figured it wasn't necessary since none of the other thumbnail generators seem to care about file size, and an empty file would automatically fail to parse in the already covered code paths. Do you want me to restore this additional check?

Reply:
Yeah it looks like only the TXT preview looks for it, so you can ignore my comment.

https://github.com/nextcloud/server/blob/64208b6d2273269cd51bb6e4a7aa939a144bc93c/lib/private/Preview/TXT.php#L26

Reply:
Oh, I already re-introduced it; I'll re-remove it 😂